### ISSUE TITLE
Apps targeting Android 12 issue

### ISSUE THREAD

**Issue by pranayansaikia** [Instamojo/instamojo-android-sdk#102]
Merging Errors: Error: android:exported needs to be explicitly specified for element <activity#com.instamojo.android.activities.PaymentDetailsActivity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.

**Comment by b8x** [comment]
is there any solution for this?? (Updated sdk or alternative?)

**Comment by aaHiHello** [comment]
<activity android:name="com.instamojo.android.activities.PaymentDetailsActivity"
            android:exported="false"
            tools:node="merge" />

**Comment by rameshvoltella** [comment]
new release also have issue when i change to 3.7.0 and sync I am getting issue
**Internal error: Artifact type null not expected, only jar or aar are handled.**
what should we should use 36.0 with the above hack? @vedhavyas @instamojosurya  .
guys help us don't be lazy in this

**Comment by mkdroid** [comment]
> new release also have issue when i change to 3.7.0 and sync I am getting issue **Internal error: Artifact type null not expected, only jar or aar are handled.** what should we should use 36.0 with the above hack? @vedhavyas @instamojosurya . guys help us don't be lazy in this

Yes getting the same issue with the latest version. Using 3.6.0 it is working fine.
